### PR TITLE
Normalization & cleaning

### PR DESCRIPTION
**Pull request by maximebiloe**

### CODE REVIEW

File: themes/StarterTheme/templates/catalog/_partials/facets.tpl [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <div id="search_filters">
   {if $activeFilters|count}
     <section class="active_filters">
-      <h1 class="h3">{l s='Active Filters'}</h1>
+      <h1 class="h3">{l s="Active Filters"}</h1>
       <ul>
         {foreach from=$activeFilters item="filter"}
-          <li>{l s='%1$s: ' sprintf=[$filter.facetLabel]} {$filter.label} <a  class="js-search-link" href="{$filter.nextEncodedFacetsURL}">{l s='Remove'}</a></li>
+          <li>{l s="%1$s: " sprintf=[$filter.facetLabel]} {$filter.label} <a  class="js-search-link" href="{$filter.nextEncodedFacetsURL}">{l s="Remove"}</a></li>
```
Comment (djfm) [inline themes/StarterTheme/templates/catalog/_partials/facets.tpl]: Are you sure $s (in %1$s) won't be interpreted as a variable here?

**Comment by kpodemski** [comment]
Why " instead of '? What is the point of this? My gosh... I already see the future and red warnings in PrestaShop validator :(

**Comment by gskema** [comment]
I too prefer single quotes in code, I keep double quotes only for html attributes.

File: themes/StarterTheme/templates/catalog/_partials/facets.tpl (hunk 1 above)
Comment (maximebiloe) [inline themes/StarterTheme/templates/catalog/_partials/facets.tpl]: Good point! I missed it

**Comment by kpodemski** [comment]
@maximebiloe @djfm  why doule quotes? why changeing something what was required for a few months in validator and where is basically no point of this?

**Comment by djfm** [comment]
As with all style considerations, there is really no specific reason to choose one over the other.

Since in HTML we use double quotes, we thought it would be nicer to have double quotes in smarty too, that's all.

The validator won't care whether you use single or double quotes and if you want to use single quotes in your themes it will be fine :)

Do you see any practical advantage to using single quotes?

**Comment by kpodemski** [comment]
Well, I don't see any practical advantage of double quotes that's why I don't understand this change :)

I'll make an screenshot because I remember that single quotes was required in validator, I really hope that will be not forced to change it once again because it is time wasting :)

All valid modules/themes from Addons has single quote and yet new starter theme will have double, this is something what I don't understand :)

**Comment by gskema** [comment]
Advantages of single quotes:
- more readable
- in JS, less confusing when used with jquery: `$("[name=\"email\"]")` vs `$('[name="email"]')`
- in PHP, doesn't prompt string interpolation, less bugs therefore
- in Smarty, probably the same advantages as PHP, also more readble

**Comment by djfm** [comment]
Ok fine, single quotes it is. But for ALL smarty arguments then :)

You can thank atom's great regexp search and replace feature :p

image

**Comment by kpodemski** [comment]
:+1:

**Pull request merged**